mousebender/simple.py

```diff
@@ -209,8 +230,18 @@ class ProjectDetails_1_3(TypedDict):
     versions: List[str]
 
 
+class ProjectDetails_1_4(TypedDict):
+    """A :class:`~typing.TypedDict` for a project details response (:pep:`792`)."""
+
+    meta: _Meta_1_4
+    name: packaging.utils.NormalizedName
+    files: list[ProjectFileDetails_1_4]
```

Comment:
`project-status` and `project-status-reason` should be here. `project-status` should be typed to the string literals listed in https://packaging.python.org/en/latest/specifications/project-status-markers/#project-status-markers . `project-status-reason` should be typed to `str`.